go/vt/schemadiff/schema_test.go

```diff
@@ -153,6 +153,7 @@ func TestNewSchemaFromQueriesLoop(t *testing.T) {
 	)
 	_, err := NewSchemaFromQueries(queries)
 	assert.Error(t, err)
+	err = UnwrapOne(err)
 	assert.EqualError(t, err, (&ViewDependencyUnresolvedError{View: "v7"}).Error())
```

Comment:
can we assert err length here as well .. to assure the numbers of errors are as expected

Reply:
`UnwrapOne()` s guaranteed to return a slice of at least one element.

Reply:
And yes! I'll add that assertion anyway.

Reply:
Rather, I'll create a `errors_test.go` with a bunch of unit tests.

Reply:
Unit tests added